packages/storybook/src/design-system/RichRadioButton/RichRadioButton.stories.tsx

```diff
@@ -0,0 +1,256 @@
+import { RichRadioButton } from '@talend/design-system';
+
+export default {
+	component: RichRadioButton,
+};
+
+export const DefaultStory = () => (
+	<RichRadioButton
+		description="Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
+	Porro nihil delectus quaerat repellat saepe officiis id aut. 
+	Culpa iste molestias optio nihil placeat magnam, odio modi earum est voluptas saepe?"
+		id={'richRadioButton'}
+		name={'richRadioButton'}
+		onChange={() => {}}
+		title="This is a title"
+	/>
+);
+
+export const RichRadioButtonWithTags = () => (
+	<RichRadioButton
+		description="Lorem ipsum dolor sit amet consectetur, adipisicing elit. 
+		Porro nihil delectus quaerat repellat saepe officiis id aut. 
+		Culpa iste molestias optio nihil placeat magnam, odio modi earum est voluptas saepe?"
+		id={'richRadioButtonWithTags'}
+		name={'richRadioButtonWithTags'}
+		onChange={() => {}}
+		tags={[
+			{
+				name: 'Tag 1',
+			},
+			{
+				name: 'Tag 2',
+				variant: 'information',
+			},
+			{
+				name: 'Tag 3',
+				variant: 'information',
+			},
+		]}
+		title="This is a title"
+	/>
```

Comment:
wdyt about this api : 

```jsx
<RichRadioButton
	id={'richRadioButtonWithTags'}
	name={'richRadioButtonWithTags'}
	onChange={() => {}}
>
	<RichRadioButton.Title>This is a title</RichRadioButton.Title>
	<RichRadioButton.Description>
		Lorem ipsum dolor <strong>sit amet consectetur, adipisicing</strong> elit. Porro nihil
		delectus quaerat <blink>repellat saepe</blink> officiis id aut. Culpa iste molestias
		optio nihil placeat magnam, odio modi earum est voluptas saepe?
	</RichRadioButton.Description>
	<Tag>Tag 1</Tag>
	<Tag.Information>Tag 2</Tag.Information>
	<Tag.Information>Tag 3</Tag.Information>
</RichRadioButton>
```

Reply:
We want to component to be strict in terms of the elements its receives